src/answers/ImagesAnswerComponent.coffee

```diff
@@ -32,13 +32,13 @@ module.exports = class ImagesAnswerComponent extends React.Component
         return
 
     # Call imageAcquirer
-    @context.imageAcquirer.acquire (id) =>
+    @context.imageAcquirer.acquire (id, rotation = 0) =>
       # Add to image list
       imagelist = @props.imagelist or []
       imagelist = imagelist.slice()
-      imagelist.push({ id: id, cover: imagelist.length == 0 })
+      imagelist.push({ id: id, cover: imagelist.length == 0 , rotation: rotation})
       @props.onImagelistChange(imagelist)
-    , (err) => throw err
+    , (err) => alert err
```

Comment:
Is err an Error object or a string? If the former, it should be alert(err.message).